tests/test-cases/Group6-VIC-Machine/6-15-Syslog.robot

```diff
@@ -25,8 +25,11 @@ ${SYSLOG_FILE}  /var/log/syslog
 *** Keywords ***
 Get Remote PID
     [Arguments]  ${proc}
-    ${pid}=  Execute Command  ps -C ${proc} -o pid=
+    #photon 4.4.152 version truncates the proc name to 15 characters in /proc/xxx/status.
+    ${proc_realname}=  Get SubString  ${proc}  0  15
```

Comment:
Will there be an exception if process name is less than 15 characters?

Reply:
Yes, if it is shorter than 16, it just copy the whole string. For example: vicadmin and vic-init used in this test are copied successfully.